frontend/src/app/actions/addon.js

```diff
@@ -21,12 +21,11 @@ function setInstalled(installed: InstalledExperiments): SetInstalledAction {
 
 export default {
   setInstalled,
+  txpInstalled: makeSimpleActionCreator("TXP_INSTALLED"),
```

Comment:
New actions for the testpilot addon install and experiment install. These trigger their popups and are distinct from enable so that manual toggling in the addon manager doesn't trigger fresh popups.